src/lib/utils/data/dataLoader.ts

```diff
@@ -3,6 +3,10 @@ import { loadMockData } from "./loadMockData"
 
 const USE_MOCK_DATA = process.env.USE_MOCK_DATA === "true"
 
+if (USE_MOCK_DATA) {
```

Comment:
nice, just noting a similar change was made in #15160 so there may be a tiny conflict here